userguide/plugins.rst

```diff
@@ -389,40 +389,93 @@ describes each required entry.
-   |                         | *226efd3a126fb86e71d60a37353d17f57af816d1c7ecad0623c21f0bf73eb0c7* |
-   |                         |                                                                    |
-   +-------------------------+--------------------------------------------------------------------+
-   | :samp:`"official":`     | Defines if this an official iXsystems supported plugin.            |
-   |                         | Enter *true* or *false*.                                           |
-   |                         |                                                                    |
-   +-------------------------+--------------------------------------------------------------------+
+   +---------------------------+--------------------------------------------------------------------+
+   | Data Field                | Description                                                        |
+   +===========================+====================================================================+
+   | :samp:`"name":`           | Name of the plugin.                                                |
+   |                           |                                                                    |
+   +---------------------------+--------------------------------------------------------------------+
+   | :samp:`"plugin_schema":`  | Optional. Enter *2* if simplified post install information has     |
+   |                           | been supplied in :file:`post_install.sh`. After specifying *2*,    |
+   |                           | echo the information to be presented to the user in                |
+   |                           | :file:`/root/PLUGIN_INFO` inside the                               |
+   |                           | :file:`post_install.sh` file.                                      |
+   |                           | See the :ref:`rslsync.json <rslsync-plugin-schema>` and            |
+   |                           | :ref:`rslsync post_install.sh <rslsync-post-install>` examples.    |
+   |                           |                                                                    |
+   +---------------------------+--------------------------------------------------------------------+
+   | :samp:`"release":`        | FreeBSD RELEASE to use for the plugin jail.                        |
+   |                           |                                                                    |
+   +---------------------------+--------------------------------------------------------------------+
+   | :samp:`"artifact":`       | URL of the plugin artifact repo.                                   |
+   |                           |                                                                    |
+   +---------------------------+--------------------------------------------------------------------+
+   | :samp:`"pkgs":`           | Port of the plugin.                                                |
```

Comment:
Not really clear what "port" means in this context.  The description does not give any idea of the format of this data.  It could mean category/portname, but a reader might think it means a port number.